jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -405,6 +405,24 @@ def test_load_spectrum_list_directory_concat(tmpdir, specviz_helper):
     assert len(specviz_helper.app.data_collection) == 41
 
 
+def test_load_2d_flux(specviz_helper):
+    spec = Spectrum1D(spectral_axis=np.linspace(4000, 6000, 10)*u.Angstrom,
```

Comment:
Maybe add comments to explain what is going on here for future devs. Hard to tell from reading the code without more context.